packages/stores/types.ts

```diff
@@ -220,7 +310,7 @@ type TUiStore = {
     is_dark_mode_on: boolean;
     is_language_settings_modal_on: boolean;
     is_mobile: boolean;
-    notification_messages_ui: JSX.Element | null;
+    notification_messages_ui: ElementType;
```

Comment:
```suggestion
    notification_messages_ui: React.FC | null;
```
@amir-deriv I think you can use `React.FC` instead 🙇🏻

Reply:
@amir-deriv For `JSX.Element ` it should give the error but `React.FC` should work 🤔

Reply:
It's the opposite right now 😄  , is there any issue with `JSX.Element` ? @farzin-deriv